src/components/ResultsMap.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<GmapMap
+		:center="centerMap"
+		:zoom="10"
+		map-type-id="terrain"
+		style="width: 100%; height: 50vh"
+	>
+		<GmapMarker
+			v-for="(m, index) in markers"
+			:key="index"
+			:position="m.geometry.location"
+			:clickable="true"
+			:draggable="false"
+			@click="centerMapToClickedLocation(m.name)"
+		/>
+	</GmapMap>
+</template>
+
+<script>
+export default {
+  props: ['searchResults'],
+  data() {
+    return {
+			markers: this.searchResults,
+			centerMap: {
+				lat: 40, lng: -105.0465427
+			}
+    }
+  },
+  methods: {
+    centerMapToClickedLocation(name) {
```

Comment:
Within the `centerMapToClickedLocation` is where we could set the name of the marked location when the user clicks on it.  Perhaps we could create a new property in `data( )` as `currentMarker` with a default of 'Select a marker on the map to identify its location', then assign it a new value within the `centerMapToClickedLocation` function.  It can be part of the HTML with the `{{ }}` syntax so that it always shows up on the map view.  

Any thoughts on this?  It is a thought for later, no need to make any changes for this PR.

Reply:
This certainly sounds like a good idea, and would also increase usability for the user. Let's talk a bit more about this in our next meeting to see how we can make this work!

Reply:
Would this same click also display the details for the park? If so maybe the default message should reference that?

Reply:
@stacyp2006 I'm thinking maybe after the user has clicked on a marker, it's name would pop up above the map.  Maybe we could have a button that takes the user to see more details about the park after its marker is selected.  Or, we could display what we generally get back from our BE (Google Places API) above our map (including the name of the place) and then have a `See More Details` button that takes them to the directions and other additional information.  Essentially, we could split up the information between what the user needs to see when the marker is clicked, and what users would want to know if they are strongly considering visiting the dog park.

Reply:
I believe my original thought from yesterday was that there would be another method that would be called within the `centerMapToClickedLocation` method that would display the details. Do you think having a 2nd method would be a second separation of concerns, or would having one handle the centering as well as the display of details?

Reply:
I think we can use [Info Window](https://developers.google.com/maps/documentation/javascript/infowindows) to display the basic info on-click and include a details button there.

Reply:
@npdarrington I think keeping that functionality separate in two functions would be good for SRP.  Invoking one inside the other is fine with me!

@bretm9 I love that idea of using the Info Window! Thanks for discovering that.  I think that would be the best way to go, especially if users are already accustomed to that kind of experience.